plugin/sampling/internal/calculationstrategy/interface.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2018 The Jaeger Authors.
```

Comment:
what is the thinking for this thing to be under `plugins`?  I can see swappable storage to be under plugins, but this seems like the core impl. I would keep it inside the collector app.

Reply:
This is core only to the adaptive sampling strategystore. Don't feel the collector app is the right place for this

Reply:
gonna go ahead and land, we can figure out where these files should live later